src/app/retrospective/sprint-activity-log/sprint-activity-log.component.html

```diff
@@ -0,0 +1,14 @@
+<ng-container>
+    <mat-list class="trails">
```

Comment:
Add custom CSS to highlight the trails and differentiate the action and the doer!!

Reply:
Done